config/coverband.rb

```diff
@@ -44,39 +44,19 @@
 
 # 3.x configuration
 Coverband.configure do |config|
-  config.root = Dir.pwd
-  config.store = if ENV['REDIS_URL']
-                   Coverband::Adapters::RedisStore.new(Redis.new(url: ENV['REDIS_URL']))
-                 else
-                   Coverband::Adapters::RedisStore.new(Redis.new)
-                 end
-
-  config.ignore = %w[vendor .erb$ .slim$]
-  # add paths that deploy to that might be different than local dev root path
-  config.root_paths = []
-
-  # reporting frequency
-  # if you are debugging changes to coverband I recommend setting to 100.0
-  # otherwise it is find to report slowly over time with less performance impact
-  # with Coverage collector coverage is ALWAYS captured this is how frequently
-  # it is reported to your back end store.
-  # NOTE: the demo site sends data for EVERY request so one can see the effects
-  # do not send 100.0 in production!
-  config.reporting_frequency = Rails.env.production? ? 10.0 : 100.0
-  config.logger              = Rails.logger
-
   # configure S3 integration
   config.s3_bucket = 'coverband-demo'
   config.s3_region = 'us-east-1'
   config.s3_access_key_id = ENV['AWS_ACCESS_KEY_ID']
   config.s3_secret_access_key = ENV['AWS_SECRET_ACCESS_KEY']
 
-  # using the new safe reload to enforce files loaded
-  config.safe_reload_files = ['config/coverband.rb']
+  # This tests a stand alone rubyscript collecting its own Coverage
+  # The script loads coverband and therefor needs to be reloaded
+  config.safe_reload_files = ['bin/script_test.rb']
```

Comment:
Random thought, we can really have some nice defaults for this. For non rails apps:

```ruby
[$0, './config/coverband.rb']
```

For rails apps:
```ruby
[$0, './config/coverband.rb', './config/application.rb', './config/boot.rb']
```

Reply:
Nice I was thinking a similar thing when I tested the script, I can add in some defaults

Reply:
I was going to pull this in but I think it might be a bit more complicated than I thought and warrant some additional testing... I am not sure if everything everyone does in application.rb would be safe to reload, could cause some methods/objects to get called twice.

Reply:
so thinking perhaps follow up with this a bit later @kbaum

Reply:
Makes sense. It's not a must have default.